tiledb/sm/array/array.cc

```diff
@@ -863,7 +897,15 @@ Array::get_enumerations_all_schemas() {
 
       // Store the loaded enumerations in the schema.
       for (auto& enmr : loaded) {
-        schema.second->store_enumeration(enmr);
+        if (!schema.second->is_enumeration_loaded(enmr->name())) {
```

Comment:
NIT: we can avoid code duplication by moving this for loop after the if-remote-else since it's the same in both cases.